src/layout/questionnaire-nav/components/nav-loop.jsx

```diff
@@ -36,64 +38,56 @@ class NavLoop extends Component {
   handleCloseComponentDetail() {
     this.setState({ showComponentModal: false });
   }
-   
-   handleDeleteComponent = () => {
-    this.props.removeComponent(this.props.editingComponentId)
+
+  handleDeleteComponent = () => {
+    this.props.removeComponent(this.props.editingComponentId);
     this.setState({ showComponentModal: false });
-  }
+  };
 
   render() {
-    const {
-      componentsStore,
-      editingComponentId,
-    } = this.props;
+    const { componentsStore, editingComponentId } = this.props;
 
-    const options =  Object.values(componentsStore)
-    .filter(component=> component.type === "LOOP")
-    .map(element => {
-     return (
-      <li onClick={() => this.handleEditComponent(element.id)}>
-        <span className="glyphicon glyphicon-menu-right" />
-        {element.nameLoop}
-      </li>
-     )
-    });
+    const options = Object.values(componentsStore)
+      .filter(component => component.type === 'LOOP')
+      .map(element => {
+        return (
+          <li onClick={() => this.handleEditComponent(element.id)}>
```

Comment:
Even if I know, this code is old, ... we should never define onClick on a li element. Can you add a focusable element ? like a button ?